src/workers/newNotificationV2Mail.ts

```diff
@@ -659,7 +658,7 @@ const worker: Worker = {
         },
         'failed to send notification email',
       );
-      counter.add(1, { channel: 'email' });
+      counters?.background?.notificationFailed?.add(1, { channel: 'email' });
```

Comment:
no meter here?

Reply:
What do you mean?

Reply:
Ah I see you removed it from everywhere now?

Reply:
Meter is what "holds" the metrics. From a meter you create counters, and I've moved the creation of all counters into `metrics.ts`. So now we just import `counters` and select the specific counter you want.

Reply:
> Ah I see you removed it from everywhere now?

Yeah

Reply:
Makes sense 👍